### ISSUE TITLE
postcss-nesting v8.0.0 – Error: [object Object] is not a PostCSS plugin

### ISSUE THREAD

**Issue by schwamic** [csstools/postcss-nesting#81]
Hi, 
I have problems getting `postcss-nesting v8.0.0.` working with PostCSS v8. ErrorMessage: `Error: [object Object] is not a PostCSS plugin`. If I use `postcss-nesting v7.0.1` the code gets compiled...

### PostCSS Config:
```
module.exports = {
  plugins: {
    'postcss-import': {},
    tailwindcss: {},
    'postcss-nesting': {},
    autoprefixer: {},
    'postcss-flexbugs-fixes': {},
    'postcss-preset-env': {},
  },
}
```

### package.json
```
  "postcss": "8.2.13",
  "postcss-flexbugs-fixes": "5.0.2",
  "postcss-import": "14.0.1",
  "postcss-nesting": "8.0.0",
  "postcss-preset-env": "6.7.0",
  "tailwindcss": "2.1.2",
```

**Comment by Ratko-Solaja** [comment]
My `postcss.config.js` - this way it is working:

```js
module.exports = {
    plugins: [
        require('postcss-import'),
        require('tailwindcss'),
        require('postcss-nesting').default,
        require('autoprefixer'),
    ]
}
```

**Comment by timfee** [comment]
Are you using Next.js? Unsure what the problem is. Trying to get the right package.json combination to repro.

https://github.com/vercel/next.js/issues/24696

**Comment by jonathantneal** [comment]
I’m in the project, fixing the package right now. I need another :30 minutes or so.

**Comment by jonathantneal** [comment]
Thanks for your patience. This was an issue in the build for CommonJS that should now be resolved in v8.0.1.

https://github.com/csstools/postcss-nesting/releases/tag/8.0.1

**Comment by schwamic** [comment]
@jonathantneal thank you very much – `postcss-nesting v8.0.1` works fine!
